src/v3/src/util/getAuthenticatorMethod.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) 2022-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import { IdxTransaction } from '@okta/okta-auth-js';
+
+export const getAuthenticatorMethod = (
+  transaction: IdxTransaction,
+): string | undefined => {
+  // currentAuthenticator is from enrollment flows and currentAuthenticatorEnrollment is from verify flows
+  // @ts-ignore OKTA-549893 TODO: Add currentAuthenticatorEnrollment to IdxContext type
+  const { context: { currentAuthenticator, currentAuthenticatorEnrollment } } = transaction;
+  const enrollMethods = currentAuthenticator?.value?.methods;
+  const verifyMethods = currentAuthenticatorEnrollment?.value?.methods;
+
+  return (enrollMethods && enrollMethods[0].type) || (verifyMethods && verifyMethods[0].type);
```

Comment:
Are there -- or will there be -- cases where we want another index? e.g., `verifyMethods[1]`. First one that comes to mind is phone voice vs sms. Maybe the index is a param with default=0?

Reply:
yeah so I was using this code in v2 as a reference: [https://github.com/okta/okta-signin-widget/blob/master/src/v2/models/AppState.ts#L63]

We dont have any cases currently where we would need another index but i agree as a util function I think we would want it to be able to get another index since there will probably be cases for it in the future.  I like having the index as a param and will change it to that. 👍